doc/sphinx/extract_rst.py

```diff
@@ -76,7 +76,7 @@
      'critters', 'lua_world_units'),
      ('data/world/immovables/artifacts/artifact00',
      'immovables', 'lua_world_units'),
-     ('data/world/resources',
+     ('data/world/resources/coal',
```

Comment:
What about the other newly created resource folders?

Reply:
Coal is the only one that contains RST documentation